sctp.c

```diff
@@ -1338,6 +1347,9 @@ void *janus_sctp_thread(void *data) {
 	}
 	JANUS_LOG(LOG_VERB, "[%"SCNu64"] Leaving SCTP association thread\n", sctp->handle_id);
 	janus_refcount_decrease(&sctp->ref);
+	if (component) {
```

Comment:
I guess that the reference count protects us here, but shouldn't you check again if `dtls->component` is NULL here, since there may have been a long while in between?

Reply:
The component is refcounted, so it should not get cleared. If it would and we would check here again, it would leak a reference.

Reply:
Good point :+1: